core/server/master/src/main/java/alluxio/master/file/LostFileDetector.java

```diff
@@ -65,7 +65,8 @@ public void heartbeat() throws InterruptedException {
       } catch (FileDoesNotExistException e) {
         LOG.debug("Exception trying to get inode from inode tree", e);
       } catch (UnavailableException e) {
-        LOG.warn("Failed to run lost file detector: {}", e.toString());
+        LOG.warn("Journal is not available. Backing off.", e);
```

Comment:
is stack trace helpful here in log warn?

Reply:
nice catch. changed back.